public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
+
+### Truffle Develop
+
+We recommend using Truffle Develop, an Ethereum client built directly into Truffle which contains an in-memory blockchain that runs only on your development machine.
+
+It processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly, and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing.
+
+Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following in a terminal:
+
+```shell
+truffle develop
+```
+
+This will run the client on `http://localhost:9545`. it will display the first 10 accounts, and a deterministic mnemonic for the blockchain. Truffle Develop uses the same mnemonic upon every launch:
```

Comment:
Needs capital `I` in "it" at beginning of second sentence.

Reply:
> a deterministic mnemonic

"Deterministic mnemonic" feels like too much detail here. I'd phrase it as the following: 

> It will display the first 10 accounts and the mnemonic used to create those accounts (for more information on account mnemonics, please see link xyz). Truffle Develop uses the same mnemonic every time to make developing your applications as easy as possible: "

I can help find a good link about mnemonics if you can't find one.

Reply:
If you have something authoritative on mnemonics, that would be appreciated!